build/notarize.js

```diff
@@ -29,7 +29,7 @@ exports.default = async function notarizing(context) {
   }
 
   const options = {
-    appBundleId: 'org.getsession.desktop',
```

Comment:
is this a required change? I am not sure I want to touch the notarizing logic,

Reply:
So the thing is our `appId` in the `package.json` is this already. The other platforms also use this id. This script is the outlier. I have confirmed it doesn't break anything on mac.

Reply:
Actually we might be able to remove this entirely since it is no longer used https://www.electron.build/configuration/mac.html#notarizelegacyoptions

Reply:
well it's on the legacy list of options which doesn't mean it's not used. We want still be on the legacy options for other things, but ok.